src/testrel.jl

```diff
@@ -554,6 +554,7 @@ function _execute_test(
         # Exec async really should return after 2-3 seconds
         headers = ["X-Request-ID" => request_id]
         exec_async(context, schema, engine, program; readtimeout=30, readonly, headers)
+        throw("uh oh")
```

Comment:
Was this done so the automated tests run? Or maybe to get a deployable image? I'm curious b/c we'll soon (hopefully) start looking at improving the developer inner loop.